variables.tf

```diff
@@ -95,8 +97,9 @@ variable "database_config" {
       deletion_protection      = false
       database_iam             = []
       enable_backup            = true
-      backup_retention         = 86400
+      backup_retention         = "86400s"
       create_db                = true
+      cron_spec_text           = "0 2 * * *"
```

Comment:
`cron_spec_text` isn't included in the (just above) object definition.  Looks like there is also a variable (below) called `cron_spec_text`?  Is/should that be used, or perhaps removed?